syntax = "proto3";

import "google/protobuf/timestamp.proto";

service GetMeterUsage {
  rpc ReturnMeterUsage (MeterUsageRequest) returns (MeterUsageReply) {}
}

message MeterUsageRequest {
  /* interestingly there is no uuid support in protobuf :( */
  string identifier = 1; 
}

message MeterUsageReply {
  message MeterUsageEntry {
    google.protobuf.Timestamp time = 1;
    float meterusage = 2;

  }
  repeated MeterUsageEntry meterusage = 1;
}
